backend/src/libs/packages/server-application/libs/helpers/create-error-response/handlers/handlers.ts

```diff
@@ -0,0 +1,13 @@
+import { type ErrorHandler } from '../types/error-handler.type.js';
+import { handleAuthErrorResponse } from './handle-auth-error-response.js';
+import { handleHttpErrorResponse } from './handle-http-error-response.js';
+import { handleValidationErrorResponse } from './handle-validation-error-response.js';
+
+const errorHandlers: ErrorHandler[] = [
```

Comment:
This should be removed after resolving the comment above, as well as find method in createErrorResponse